### ISSUE TITLE
clusteringEnabled does not change dynamically

### ISSUE THREAD

**Issue by Adam-Schlichtmann** [venits/react-native-map-clustering#176]
I have clusteringEnabled set to true initially and change it based off of a state value. 

The reasoning behind this is if a marker is selected a carousel appears in the bottom of my map to show more information about the marker. In order to let users see and click on all markers if one is selected I want the clusters to disappear. 

When the clusteringEnabled is changed to false, all of the markers appear, but the ClusteredMarker is still showing over the markers it is representing.

I do have a work around for now: 
```
clusteringEnabled={showClusters}
renderCluster={({
            onPress,
            clusterColor,
            clusterTextColor,
            clusterFontFamily,
            ...marker
          }) =>
            showClusters ? (
              <ClusteredMarker
                key={`cluster-${marker.id}`}
                {...marker}
                onPress={onPress}
                clusterColor={clusterColor}
                clusterTextColor={clusterTextColor}
                clusterFontFamily={clusterFontFamily}
                tracksViewChanges={false}
              />
            ) : null
```

**Comment by ravenariola04** [comment]
Is this fixed already? I really need this props and its not working as well @Adam-Schlichtmann .

**Comment by arunkmoury** [comment]
Is there any solution available?

**Comment by venits** [comment]
Fixed in version `3.4.0`
